extension/src/panel.ts

```diff
@@ -285,15 +279,15 @@ export class HistoryDebuggerPanel {
         )
       }
     } catch (err: any) {
-      if (err?.code === "FileNotFound") {
+      if (err?.code === vscode.FileSystemError.FileNotFound.name) {
         if (!configuredAbsolutePath && (vscode.workspace.workspaceFolders?.length ?? 0) > 1) {
           throw new Error(
             `Configured temporal.replayerEndpoint (${replayerEntrypoint}) not found (multiple workspace folders found, consider using an absolute path to disambiguate)`,
           )
```

Comment:
I looked into adding `cause` to the `Error` here, but it looks like it isn't something we can update on our end. 

The types used in [lib.es5.d.ts](https://github.com/microsoft/TypeScript/blob/main/lib/lib.es5.d.ts#L1045) don't include `cause` for the `ErrorConstructor`. We need [lib.es2022.d.ts](https://github.com/microsoft/TypeScript/blob/v4.7.4/lib/lib.es2022.error.d.ts#L29) and later versions, but the vscode [lib compiler option](https://github.com/microsoft/vscode/blob/1fae59815fe45065718e9d5ae7eb335c9cfba07e/src/tsconfig.vscode-dts.json#L16) is currently set to `es5`.

Reply:
If you can make it work it would be nice, but not critical.